src/main/java/io/opentracing/contrib/jdbc/JdbcTracingUtils.java

```diff
@@ -88,7 +92,7 @@ static void onError(Throwable throwable, Span span) {
   }
 
   private static Map<String, Object> errorLogs(Throwable throwable) {
-    Map<String, Object> errorLogs = new HashMap<>(2);
+    Map<String, Object> errorLogs = new HashMap<>(3);
```

Comment:
why it is changed from 2 to 3?

Reply:
Avoid expansion of capacity

Reply:
if we add only 2 items, why capacity is 3?

Reply:
Because the default load factor is 0.75